content/blocksPages/astro.json

```diff
@@ -1,10 +1,23 @@
 {
```

Comment:
Why do we have the Astro edits on this branch too?

Reply:
Snuck in from my other PR, removed it from this one.